oat-sa/extension-tao-itemqti
# fix/TR-3025/refactor-hottext-rtl-support

**Related to:** [TR-3025](https://oat-sa.atlassian.net/browse/TR-3025)

**Description:**
There is an issue when hottext are at the end of a line. It floats to the following line because it needs more space than simple text and creates a block due to the `display: flex`
The `display: flex` is not needed when the text language is an RTL language. So I remove it.

**Changes:**

- Remove CSS rule to `display: flex` on RTL mode
- Refactor margin and padding to simplify CSS rules

**How to check:**

- Create an item with a Hottext interaction with **large** LTR text (e.g.: English) and select some words for the interaction. Select words at the beginning and the end of lines to check if the issue is fixed.
-  Create an item with a Hottext interaction with **large** RTL text (e.g.: Arabic, use **Google translator to use Arabic text**) and select some words for the interaction. Select words at the beginning and the end of lines to check if the issue is fixed. Don't forget to change the Language of the item on AUT. 
- Review the test on LTR and RTL mode.

**TAE Environment:**

- Link to TAE environment and credentials on the comments of the ticket

**Local test:**

- Link to composer file on the comments of the ticket and a test attached with Arabic item.
